src/components/wallet/UnlockForm.style.tsx

```diff
@@ -0,0 +1,90 @@
+import AF from 'antd/lib/form'
+import AT from 'antd/lib/typography/Text'
+import styled from 'styled-components'
+import { palette } from 'styled-theme'
+
+import { media } from '../../helpers/styleHelper'
+import BaseButton from '../uielements/button'
+
+export const Form = styled(AF)`
+  display: flex;
+  flex-direction: column;
+
+  ${media.lg`
+    height: 100%;
+  `}
+
+  .ant-form-item {
+    width: 100%;
+    margin: 0;
+  }
+`
+
+export const Header = styled('div')`
+  display: flex;
+  justify-content: center;
+  position: relative;
+
+  .ant-typography {
```

Comment:
Is there another way to style fonts here? Accessing `antd`  style selectors such as `.ant-typography` are always unsafe. I know we already have that in different places, mostly by adapting styles from BEPSwap.